### PR TITLE
:WIP: Add upgrade tests in feature test phony

### PR DESCRIPTION
**Pull request by kashifest**
Add upgrade tests in feature test phony

### CODE REVIEW

**Comment by maelk** [comment]
/approve
but you should run the feature tests on this PR too

**Comment by kashifest** [comment]
> /approve
> but you should run the feature tests on this PR too

ok.

**Comment by kashifest** [comment]
hold until the upgrade tests are initialized properly

File: Makefile [hunk 1]
```diff
@@ -41,6 +41,6 @@ remediation_test:
 upgrade_test:
 	make -C ./scripts/feature_tests/upgrade/
 
-feature_tests: setup_env remediation_test cleanup_env pivoting_test
+feature_tests: setup_env remediation_test cleanup_env upgrade_test cleanup_env pivoting_test
```
Comment (Jaakko-Os) [inline Makefile:44]: there's a new target for upgrade so not included in 'feature_tests'
implemented in https://github.com/metal3-io/metal3-dev-env/pull/390

**Comment by kashifest** [comment]
@Jaakko-Os I think this PR is obsolete now right?

**Comment by kashifest** [comment]
Closing this PR since #390 has added upgrade test in a separate make target

**Pull request closed without merging**